docs/src/modules/components/ApiPage/sections/ToggleDisplayOption.tsx

```diff
@@ -8,10 +8,10 @@ import TableChartRoundedIcon from '@mui/icons-material/TableChartRounded';
 import ReorderRoundedIcon from '@mui/icons-material/ReorderRounded';
 import useEnhancedEffect from '@mui/utils/useEnhancedEffect';
 
-type ApiDisplayOptions = 'collapsed' | 'expended' | 'table';
+type ApiDisplayOptions = 'collapsed' | 'expanded' | 'table';
 
-const options: ApiDisplayOptions[] = ['collapsed', 'expended', 'table'];
-const DEFAULT_LAYOUT: ApiDisplayOptions = 'expended';
+const options: ApiDisplayOptions[] = ['collapsed', 'expanded', 'table'];
+const DEFAULT_LAYOUT: ApiDisplayOptions = 'expanded';
```

Comment:
This typo fix might need extra work.

In `getOption` and `getRandomOption` the loclastorage can still return `'expended'`.

Reply:
Right, indeed. I'll make sure it uses a default value when local storage has something unknown.